data/tlg0010/tlg001/__cts__.xml

```diff
@@ -5,9 +5,9 @@
       
    <ti:edition projid="greekLit:perseus-grc2" urn="urn:cts:greekLit:tlg0010.tlg001.perseus-grc2" workUrn="urn:cts:greekLit:tlg0010.tlg001" xml:lang="grc">
         
-      <ti:label xml:lang="eng">Against Euthynus</ti:label>
+      <ti:label xml:lang="grc">προς ευθύνουν</ti:label>
         
-      <ti:description xml:lang="eng">Isocrates. Isocrates with an English Translation in three volumes, by Larue Van Hook, Ph.D., LL.D. Cambridge, MA, Harvard University Press; London, William Heinemann Ltd. 1945-1968.</ti:description>
+      <ti:description xml:lang="eng">Isocrates, vol. 3. Van Hook, Larue, editor. Cambridge, MA, Harvard University Press; London, William Heinemann Ltd., 1945 (printing).</ti:description>
```

Comment:
capitalize Vol.?

Reply:
I can't remember how we have done that throughout the files, but I will capitalize it here, maybe something to standarize later.